usecase/usecase-wrt-linux-tests/samples/WebNotification/README.md

```diff
@@ -0,0 +1,9 @@
+## Usecase Design
+
+This sample demonstrate Web Notification feature basic functionalities, include:
+
+* Notification serviceWorker register, showNotification and notificationclick
```

Comment:
At https://crosswalk-project.org/jira/browse/XWALK-3654, Crosswalk makes a workaround of the Web Notifications implementation. 'Desktop' notification doesn't implement the latest push and service worker based notification. It should be based on W3C spec, but not the WHATWG spec. Please double check it.